app/views/lists/show.html.erb

```diff
@@ -43,6 +56,10 @@
   </div>
 <% end %>
 
+<% if flash[:item_notice] %>
+  <%= render 'errors', errors: flash[:item_notice] %>
+<% end %>
+
 <%= link_to lists_path, class: 'button' do %>
```

Comment:
So it's more common to use flash messages for something like "List item created" or "We couldn't add your item because there were errors" rather than for the specific validation errors. We would just use `@list.errors` or `@item.errors` at this point. Does that make sense?

Reply:
Had a chat this morning, potentially could use Action Cable to broadcast the errors, we were unable to use @list.errors or @item.errors as the page is a 'single page app' with no form postbacks. I think this is something we can come back to.

Reply:
I see your point, however I wasn't sure how to access the correct variables given that I've worked in a more SPA type style. e.g., in my index page (`ListsController.index`) I only have access to `@lists`, not a new list `@list` from which I can pick up the errors.  

Tim suggested looking into ActionCable, I'll check it out, but it would be great to talk about this when you're back @mrGrazy .